ko_KR/readme.md

```diff
@@ -0,0 +1,971 @@
+	await promiseFn();
+});
+
+test.after(t => {
+	return new Promise(/* ... */);
+});
+
+test.cb.beforeEach(t => {
+	setTimeout(t.end);
+});
+
+test.afterEach.cb(t => {
+	setTimeout(t.end);
+});
+```
+
+참고로 `beforeEach`와 `afterEach` 그냥 테스트의 이전과 이후에 각각 실행되며 기본적으로 테스트를 동시에 실행시킵니다. 만약 각 테스트를 위해 전역 상태를 설정해야 할 필요가 있는 경우 ([예시](https://github.com/sindresorhus/ava/issues/560)와 같이 `console.log`를 감시한다거나), 테스트가 확실하게 [직렬화로 실행](#테스트를-직렬로-실행하기)하는지 확인해야 합니다.
+
+참고로 AVA는 각 테스트 파일을 분리된 프로세스에서 실행합니다. 따라서 `after`-훅이 바로 프로세스가 종료되기 전에 실행되는 이상 따로 전역 상태를 정리하지 않아도 됩니다.
+
+`beforeEach` & `afterEach` 훅은 다음과 같이 컨텍스트를 테스트와 공유할 수 있습니다:
+
+```js
+test.beforeEach(t => {
+	t.context.data = generateUniqueData();
+});
+
+test(t => {
+	t.is(t.context.data + 'bar', 'foobar');
+});
+```
+
+기본적으로 `t.context`는 객체이지만 다시 할당할 수 있습니다:
+
+```js
+test.beforeEach(t => {
+	t.context = 'unicorn';
+});
+
+test(t => {
+	t.is(t.context, 'unicorn');
+});
+```
+
+`before`와 `after` 훅 사이의 컨텍스트 공유는 할 수 *없습니다*.
+
+### 테스트 한정자 체이닝하기
+
+`.serial`, `.only` 그리고 `.skip` 한정자는 순서에 상관없이, `test`, `before`, `after`, `beforeEach` 그리고 `afterEach`와 같이 사용할 수 있습니다. 예시는 다음과 같습니다:
+
+```js
+test.before.skip(...);
+test.skip.after(...);
+test.serial.only(...);
+test.only.serial(...);
+```
+
+이 말은 임시적으로 `.skip` 또는 `.only`를 테스트의 끝에 추가하거나 훅 선언에서 별 다른 수정 없이 사용할 수 있다는 말입니다.
+
+### 커스텀 가정
+
+내장에서 제공하는 가정 라이브러리 대신, 또는 추가적으로 가정 실패시 예외를 던지는것을 지원한다면, 어느 가정 라이브러리라도 사용할 수 있습니다.
+
+이는 [빌트-인 가정](#가정)을 사용하는 것과는 달리 좋은 경험을 제공하지 못하며 [가정 계획](#가정-계획하기)을 사용할 수 없습니다. ([이슈 #25](https://github.com/sindresorhus/ava/issues/25)를 참고하세요)
+
+```js
+import assert from 'assert';
+
+test(t => {
+	assert(true);
+});
+```
+
+### ES2015 지원
+
+AVA는 [Babel 6](https://babeljs.io)를 통한 ES2015를 내부적으로 기본 지원합니다. 그저 테스트 코드를 ES2015로 작성하면 됩니다. 다른 추가 작업은 필요하지 않습니다. 또한 어떠한 Babel 버전도 프로젝트 내에서 사용할 수 있습니다. 내부에서 자체적으로 [`es2015`](https://babeljs.io/docs/plugins/preset-es2015/)와 [`stage-2`](https://babeljs.io/docs/plugins/preset-stage-2/) 프리셋을 사용함과 동시에 [`espower`](https://github.com/power-assert-js/babel-plugin-espower)와 [`transform-runtime`](https://babeljs.io/docs/plugins/transform-runtime/) 플러그인을 사용합니다.
+
+AVA에서 사용하는 Babel 구성은 다음과 같습니다:
+
+```json
+{
+  "presets": [
+    "es2015",
+    "stage-2"
+  ],
+  "plugins": [
+    "espower",
+    "transform-runtime"
+  ]
+}
+```
+
+AVA의 [`package.json` 구성](#구성)에서 `babel` 옵션을 통해 AVA 테스트 파일의 트랜스파일 동작을 커스터마이즈 할 수 있습니다. 예를 들어 프리셋을 덮어쓰려면 다음과 같이 설정할 수 있습니다:
+
+```json
+{
+	"ava": {
+		 "babel": {
+			 "presets": [
+					"es2015",
+					"stage-0",
+					"react"
+			 ]
+		 }
+	},
+}
+```
+
+또한 특별한 `"inherit"` 키워드를 사용할 수 있습니다. 이 키워드는 AVA가 프로젝트 내의 [`.babelrc` 또는 `package.json` 파일](https://babeljs.io/docs/usage/babelrc/)을 사용하도록 구성을 연기합니다. 이 방법을 사용하면 AVA를 위해 추가적인 설정을 할 필요 없이 소스 파일에 대한 구성을 그대로 사용하여 테스트 파일을 트랜스파일 할 수 있습니다:
+
+ ```json
+{
+	"babel": {
+		"presets": [
+			"es2015",
+			"stage-0",
+			"react"
+		]
+	},
+	"ava": {
+		"babel": "inherit"
+	},
+}
+```
+더 많은 예시와 자세한 구성 옵션에 대한 설명은 AVA의 [`.babelrc` 레시피](docs/recipes/babelrc.md)를 참고하세요.
```

Comment:
Please note this link does not exist yet in Korean, it's best to link to the English version : `(https://github.com/sindresorhus/ava/blob/master/docs/recipes/babelrc.md)`